lib/project_types/script/layers/infrastructure/assemblyscript_project_creator.rb

```diff
@@ -90,7 +48,8 @@ def write_package_json
                 "assemblyscript": "^0.14.0"
               },
               "scripts": {
-                "test": "asp --config test/as-pect.config.js --summary --verbose"
+                "test": "asp --summary --verbose",
+                "build": "npx shopify-scripts-toolchain-as build --src src/script.ts --binary #{script_name}.wasm -- --lib node_modules --optimize --use Date="
```

Comment:
Once we start tackling work for the language server, I place -- hopefully most -- of this in an `asconfig.json` file. That work might happen next cycle. This will probably be reduced to 

```
npx shopify-scripts-toolchain-as build --binary foo.wasm
```

For that, we also need to make the toolchain stop requiring the `src` option, since it can be defined in the `asconfig.json` file. I'm not doing that in this PR in the hope of keeping the changes small.